torch_geometric/utils/get_mesh_laplacian.py

```diff
@@ -52,58 +52,66 @@ def get_mesh_laplacian(
     assert pos.size(1) == 3 and face.size(0) == 3
 
     num_nodes = pos.shape[0]
+    batch_size = pos.shape[0] // 3

```

Comment:
Can you clarify this for me? Why is `batch_size` the number of nodes divided by 3?